src/js/main.js

```diff
@@ -84,4 +89,83 @@ Site.Paypal = {
 
 };
 
+Site.Portraits = {
+  init: function() {
+    var _this = this;
+
+    // Get filters form element
+    _this.$form = $('#portraits-filters');
+
+    // Get filters select elements
+    _this.$filters = _this.$form.find('select');
+
+    // Init grid
+    _this.initGrid();
+
+    // Bind select filters
+    _this.$filters = $('.filter-select').on('change', _this.handleFilterChange.bind(_this));
+
+  },
+
+  initGrid: function() {
+    var _this = this;
+
+    // Set grid with isotope
+    _this.$grid = $('#portraits-grid').isotope({
+      // options
+      itemSelector: '.grid-item',
+      layoutMode: 'fitRows'
+    });
+
+    // Workaround to make it compatible with lazysizes
+    _this.$grid[0].addEventListener('load', (function(){
+      var runs;
```

Comment:
we should try to just comment the link to the source when we use snippets like this imo